github-stats/mysql/README.md

```diff
@@ -0,0 +1,27 @@
+Set-up Google Cloud SQL Proxy
+-----------------------------
+
+Create the database, from the console (recommended) or from command line:
+```
+gcloud sql instances create github-database
+```
+
+Fetch Service Account credentials and save it as "credential.json"
+https://developers.google.com/identity/protocols/application-default-credentials
```

Comment:
You can link to https://cloud.google.com/storage/docs/authentication#service_accounts which has a decent walkthrough for what to do. The tl;dr is

1. Create or use an existing service account you will use. You could use the default compute service account, but their doc page recommends creating one specifically for the proxy.
2. Grant service account required permissions
3. Generate a JSON key for the service account (there's a dropdown for this)
4. Download key to local host so you can create a kubectl secret with the key.

Only issue is I'm not sure what permissions are required by the proxy. I checked https://cloud.google.com/sql/docs/sql-proxy#invocations and they don't seem to specify what role(s) the service account requires, so I filed a bug, hopefully we figure that out. I suspect `compute network admin` should work. You can experiment.